packages/skill-lesson/markdown/markdown-components.tsx

```diff
@@ -0,0 +1,47 @@
+import {MuxPlayerRefAttributes} from '@mux/mux-player-react/*'
+import {hmsToSeconds} from '@skillrecordings/time'
+import {RefObject} from 'react'
+import ReactMarkdown, {Components} from 'react-markdown'
+
+const getMarkdownComponents = ({
```

Comment:
beginning of PT components replacement. so far I got timestamps working for transcripts.